### PR TITLE
0.8.2 post release

### PR DESCRIPTION
**Pull request by tonyhoo**
*Issue #, if available:*

*Description of changes:*

Update the stable website for 0.8.2 release

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Innixma — CHANGES_REQUESTED** [review]
We actually shouldn't make these changes. The problem is not the version in stable, it is the fact that it is pointing to the wrong link.

**Comment by Innixma** [comment]
Opened a PR which should fix the problem: https://github.com/autogluon/autogluon/pull/3553

**Comment by tonyhoo** [comment]
@Innixma We need to update the version for stable anyway as it is still reflecting the 0.8.1 release which is outdated unless we have a reason to do so

**Comment by gidler** [comment]
These look like the correct changes to me. I'm wondering if we missed [these steps](https://github.com/autogluon/autogluon/blob/master/release_instructions/ReleaseInstructions.md#post-release) during the post-release of version `0.8.2`. Maybe that means we need to make them more explicit?

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.8.2
\ No newline at end of file
+0.8.3
```
Comment (Innixma) [inline VERSION:1]: This should be 0.8.2 for the stable release

File: docs/conf.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 sys.path = ['.', '..'] + sys.path
 
 project = 'AutoGluon'
-release = '0.8.2'
+release = '0.8.3'
```
Comment (Innixma) [inline docs/conf.py:7]: This should be 0.8.2 for the stable release

**Review by Innixma — APPROVED** [review]
Updated to address requested changes

**Pull request merged**